### PR TITLE
Cycle KubeCon info on landing page

### PR DESCRIPTION
**Pull request by ialidzhikov**
Ref #15170.

### CODE REVIEW

File: content/es/_index.html [hunk 1]
```diff
@@ -44,12 +44,12 @@ <h2>El desafío de migrar más de 150 microservicios a Kubernetes</h2>
         <br>
         <br>
         <br>
-        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019" button id="desktopKCButton">Asista a la KubeCon en Shanghai el 24-26 de Junio</a>
+        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019" button id="desktopKCButton">Asista a la KubeCon en San Diego el 18-21 de Nov. 2019</a>
```
Comment (electrocucaracha) [inline content/es/_index.html]:
```suggestion
        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019" button id="desktopKCButton">Asista a la KubeCon en San Diego del 18 al 21 de Nov. 2019</a>
```

File: content/es/_index.html [hunk 2]
```diff
@@ -44,12 +44,12 @@ <h2>El desafío de migrar más de 150 microservicios a Kubernetes</h2>
...
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019" button id="desktopKCButton">Asista a la KubeCon en San Diego el 18-21 de Nov.</a>
+        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2020/" button id="desktopKCButton">Asista a la KubeCon en Amsterdam el 30 Marzo-2 Abr. 2020</a>
```
Comment (electrocucaracha) [inline content/es/_index.html]:
```suggestion
        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2020/" button id="desktopKCButton">Asista a la KubeCon en Amsterdam del 30 Marzo al 2 Abril 2020</a>
```

**Comment by ialidzhikov** [comment]
Applied the suggestions. PTAL.

**Comment by sftim** [comment]
This will need approvals from all the localizations it touches.
It's good to keep things up to date I think, so:
/priority important-soon

**Comment by raelga** [comment]
:+1: to the Spanish localization.

**Pull request merged**